docs/UserGuide.md

```diff
@@ -5,36 +5,45 @@ title: User Guide
 
 AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
 
-* Table of Contents
-{:toc}
+## Table of contents
+
+- [Quickstart](#Quickstart)
+- [Features](#Features)
+    - [Viewing help](#Viewing-help-help)
+    - [Adding clients](#Adding-a-client-client-add)
+    - [Viewing clients](#Viewing-clients-client-view)
+    - [Finding clients](#Finding-clients-client-find)
+    - [Updating clients](#Updating-a-client-client-update)
+    - [Deleting clients](#Deleting-a-client-client-delete)
+    - [Saving data](#Saving-data)
+    - [Adding notes](#Adding-notes-for-a-country)
+    - [Deleting notes](#Deleting-a-client-client-delete)
+    - [Updating client notes](#Updating-client-notes-client-note-update)
+    - [Filtering by country](#Filtering-clients-by-country-country-filter)
+    - [Adding country notes](#Adding-notes-for-a-country-country-note)
+    - [Clearing all entries](#Clearing-all-entries-clear)
+    - [Exiting the program](#Exiting-the-program--exit)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## Quickstart
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java 11
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the application file (.jar file) from [here](#https://github.com/AY2021S1-CS2103T-F11-4/tp/releases)
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Double-click the file to start the app
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+4. Type any command in the command box and press Enter to execute it. e.g. typing [`help`](#Viewing-help-help) and
+pressing Enter will open the help window
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+5. (Recommended) The [`help`](#Viewing-help-help) command will show a quick page of some commonly used commands to get
+you started in using TBM
```

Comment:
There's inconsistency between this and the description of the help command below. Also, I think the help contents will be opened in a window and not a page?